clients/admin-ui/src/features/dataset/DatasetCollectionView.tsx

```diff
@@ -0,0 +1,71 @@
+import { Box, Select, Spinner } from "@fidesui/react";
+import { ChangeEvent, useEffect, useState } from "react";
+
+import { FidesKey } from "../common/fides-types";
+import { useGetDatasetByKeyQuery } from "./dataset.slice";
+import DatasetFieldsTable from "./DatasetFieldsTable";
+import { DatasetCollection } from "./types";
+
+const useDataset = (key: FidesKey) => {
+  const { data, isLoading } = useGetDatasetByKeyQuery(key);
+
+  return {
+    isLoading,
+    dataset: data,
+  };
+};
+
+interface Props {
+  fidesKey: FidesKey;
+}
+
+const DatasetCollectionView = ({ fidesKey }: Props) => {
+  const { dataset, isLoading } = useDataset(fidesKey);
+  const [activeCollection, setActiveCollection] = useState<
+    DatasetCollection | undefined
+  >();
```

Comment:
I think you can initialize this as an empty array by default so you don't need to do the if/else logic below `useState<DatasetCollection | []>([])`

Reply:
so in line with the comment above about `DatasetCollection` being an object, I'll leave this one as-is since it shouldn't be an array